### ISSUE TITLE
The “issue template” link is broken in the contributing page

### ISSUE THREAD

**Issue by sanchezy** [alan-turing-institute/the-turing-way#846]
### Summary

In the [Contributing page](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md) guidelines the link to access the issue template is broken. I was told there are two new templates, so I propose to add the correct links with an explanation of when to use each of the new templates.

### What needs to be done?

- [ ] *Update the links. There are two new templates, the correct link is [here](https://github.com/alan-turing-institute/the-turing-way/issues/new/choose)*
- [ ] *Add an explanation on when to use each of the new templates*

### Who can help?

*  I'll take this on or anyone else with experience on Github

---

### Updates

* @erictleung made changes in his fork, waiting to see if I can add something else or the bug is fixed already

**Comment by malvikasharan** [comment]
Ah, thanks for issue. I am working on it.

**Comment by erictleung** [comment]
@sanchezy FYI, I've noticed that someone else has created an issue and pointed this out [here](https://github.com/alan-turing-institute/the-turing-way/issues/794).

I noticed too late that you've wanted to make these changes. I made changes on my fork. Feel free to use them if you want to make these changes

https://github.com/erictleung/the-turing-way/commit/1347b1c357c3e093d1f7a60f23f7fb78d85cb077
